engine/src/main/java/org/hibernate/search/engine/metadata/impl/BridgeDefinedField.java

```diff
@@ -24,6 +27,13 @@
 	 */
 	private final String absoluteName;
 	private final FieldType type;
+	private Map<Class, Object> extra = new HashMap<>();
```

Comment:
Shouldn't this be `Map<Class<?>, Object>` instead? It will save us one compiler warning.